### PR TITLE
[FIX] merge_records: _adjust_merged_values_orm

### PR DESCRIPTION
**Pull request by mreficent**
Two fixes.

### CODE REVIEW

**Comment by mreficent** [comment]
:green_apple:

File: openupgradelib/openupgrade_merge_records.py [hunk 1]
```diff
@@ -334,7 +334,7 @@ def _adjust_merged_values_orm(env, model_name, record_ids, target_record_id,
             if op == 'merge':
                 if not getattr(target_record, field.name) and l and not \
                         vals.get(field.name):
-                    vals[field.name] = l[:1]
+                    vals[field.name] = [x for x in l if x][:1]
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade_merge_records.py]: You should filter this before here, for having proper comparison on `if` sentence.

Comment (mreficent) [reply openupgradelib/openupgrade_merge_records.py]: done

File: openupgradelib/openupgrade_merge_records.py [hunk 2]
```diff
@@ -332,14 +332,19 @@ def _adjust_merged_values_orm(env, model_name, record_ids, target_record_id,
...
+                l = [x for x in l if x]
                 if not getattr(target_record, field.name) and l and not \
                         vals.get(field.name):
                     vals[field.name] = l[:1]
     # Curate values that haven't changed
     new_vals = {}
     for f in vals:
-        if vals[f] != getattr(target_record, f):
-            new_vals[f] = vals[f]
+        if fields[f].type != 'many2many':
```
Comment (pedrobaeza) [inline openupgradelib/openupgrade_merge_records.py]: This line is failing:

https://travis-ci.org/OCA/OpenUpgrade/builds/460391918#L7111

Comment (mreficent) [reply openupgradelib/openupgrade_merge_records.py]: my bad 🙁

Comment (mreficent) [reply openupgradelib/openupgrade_merge_records.py]: Fixed in https://github.com/OCA/openupgradelib/pull/132

**Pull request merged**